composer/trainer/trainer.py

```diff
@@ -660,6 +660,7 @@ class Trainer:
             Example 2: ``load_exclude_algorithms = ["FusedLayerNorm", "Alibi"]`` would exclude FusedLayerNorm and Alibi from loading.
 
             (default: ``None``)
+        load_planner (LoadPlanner, optional): A :class:`~torch.distributed.checkpoint.planner.LoadPlanner`. (default: ``None``)
```

Comment:
```suggestion
        load_planner (Optional[LoadPlanner], optional): A :class:`~torch.distributed.checkpoint.planner.LoadPlanner`. (default: ``None``)
```

Reply:
Resolving but will update based on conclusion of above thread